### DISCUSSION TITLE
can't start latest container image.

### DISCUSSION THREAD

**Discussion by cyrrelsneer** [lloesche/valheim-server-docker#211] [Q&A] [answered]
Hi,

after this mornings update (got a cronjob for that), my container won't come up anymore:
"Failure: OCI runtime create failed: container_linux.go:367: [...]"

Anyone else with this problem?

**Comment by lloesche** [comment]
I am assuming you are on Synology. Check #208
Synology seems to download the new image but uses the old image's CMD. The fix is to delete the container and recreate it from the new image. All your world data and server downloads are stored outside the container so aren't affected.

**Reply by cyrrelsneer** [reply]
Hi,

I have absolutely no idea how "Synology" came into the headline...
Sorry for that - I was either in thoughts or I don't know what happened.

It's running on an x64 supermicro server in docker ;-)

And recreating the container with pulling the latest image is giving me the same error

**Reply by lloesche** [reply]
How are you running the container? On Synology the issue is caused by Synology remembering the old container's CMD instead of using the one specified in the new image.
6 days ago the CMD changed from
```
CMD ["/usr/bin/supervisord", "-c", "/etc/supervisor/supervisord.conf"]
```
to
```
CMD ["/usr/local/sbin/bootstrap"]
```

So if anywhere in your config you see the old one staring in `/usr/bin/supervisord`, replace it with the second one `/usr/local/sbin/bootstrap`.

**Reply by cyrrelsneer** [reply]
I am running the container on  Ubuntu 20.04 on my supermicro.
I am using Portainer for managing docker, recreating didn't work.
Right now I have deleted the container and created it again from scretch (if you can say so) and it seems to be running again.

Thanks for the inspiration, that helped a lot!

**Comment by lloesche** [comment, marked as answer]
@podbiker has created [an updated section in the README](https://github.com/lloesche/valheim-server-docker#changing-startup-cmd-in-portainer) that explains how to change the startup CMD in Portainer without having to recreate the container from scratch.
